management/prompt.go

```diff
@@ -0,0 +1,24 @@
+package management
+
+type PromptSettings struct {
+	// Which login experience to use. Can be `new` or `classic`.
+	UniversalLoginExperience string `json:"universal_login_experience,omitempty"`
+}
+
+type PromptManager struct {
+	m *Management
+}
+
+func NewPromptManager(m *Management) *PromptManager {
+	return &PromptManager{m}
+}
+
+func (pm *PromptManager) GetSettings() (*PromptSettings, error) {
```

Comment:
Can we rename these operations to `Read` and `Update`? Similar to the `Tenant` or `Branding` resources.